.gitmodules

```diff
@@ -64,7 +64,7 @@
 	url = https://github.com/GNOME/libxml2.git
 [submodule "contrib/libgsasl"]
 	path = contrib/libgsasl
-	url = https://github.com/ClickHouse-Extras/libgsasl.git
+	url = https://github.com/oandrew/libgsasl.git
```

Comment:
make sure to change back once https://github.com/ClickHouse-Extras/libgsasl/pull/2 is merged